_includes/partners.html

```diff
@@ -1,57 +1,66 @@
-            <div class="col-sm-4 col-xs-6 sponsor">
-                <a href="https://www.anaconda.com/">
-                    <img src="{{site.url}}/assets/anaconda-logo-colors.svg" class="company-logo" height="60" width="200" alt="anaconda inc logo">
-                </a>
-            </div>
-            <div class="col-sm-4 col-xs-6 sponsor">
-                <a href="https://www.bloomberg.com/">
-                    <img src="{{site.url}}/assets/bloomberg-color.svg" class="company-logo" alt="bloomberg logo">
-                </a>
-            </div>
-            <div class="col-sm-4 col-xs-6 sponsor">
-                <a href="https://techblog.netflix.com">
-                    <img src="{{site.url}}/assets/netflix-color.svg" class="company-logo" alt="netflix logo">
-                </a>
-            </div>
-            <div class="col-sm-4 col-xs-6 sponsor">
+    </div>
+    <div class="row">
+        <div class="col-sm-3 col-xs-6 sponsor">
+            <a href="https://www.anaconda.com/">
+                <img src="{{site.baseurl}}/assets/anaconda-logo-colors.svg" class="company-logo" height="60" width="200" alt="anaconda inc logo">
+            </a>
+        </div>
+        <div class="col-sm-3 col-xs-6 sponsor">
+            <a href="https://www.bloomberg.com/">
+                <img src="{{site.baseurl}}/assets/bloomberg-color.svg" class="company-logo" alt="bloomberg logo">
+            </a>
+        </div>
+        <div class="col-sm-3 col-xs-6 sponsor">
+            <a href="https://techblog.netflix.com">
+                <img src="{{site.baseurl}}/assets/netflix-color.svg" class="company-logo" alt="netflix logo">
+            </a>
+        </div>
+        <div class="col-sm-3 col-xs-6 sponsor">
                 <a href="https://www.calpoly.edu/">
-                    <img src="{{site.url}}/assets/poly-color.svg" class="company-logo" alt="cal poly logo">
-                </a>
-            </div>
-            <div class="col-sm-4 col-xs-6 sponsor">
-                <a href="https://www.berkeley.edu/">
-                    <img src="{{site.url}}/assets/berkeley-color.svg" class="company-logo" alt="UC Berkeley logo">
-                </a>
-            </div>
-            <div class="col-sm-4 col-xs-6 sponsor ">
-                <a href="https://quantstack.net/">
-                    <img src="{{site.url}}/assets/quantstack-color.svg" class="company-logo" alt="QuantStack logo">
+                    <img src="{{site.baseurl}}/assets/poly-color.svg" class="company-logo" alt="cal poly logo">
                 </a>
-            </div>
-            <div class="col-sm-4 col-xs-6 sponsor ">
-                <a href="https://www.jpmorganchase.com/">
-                    <img src="{{site.url}}/assets/jpmc_logo.png" class="company-logo" alt="JPMorgan Chase logo">
-                </a>
-            </div>
-            <div class="col-sm-4 col-xs-6 sponsor">
+        </div>
+    </div>
+    <div class="row">
```

Comment:
Wraping in `row`s won't work on XS, in here you are assuming there are 4 logo per line; though on narrow screen there are only two. 
![Screen Shot 2019-03-20 at 3 27 15 PM](https://user-images.githubusercontent.com/335567/54723399-ac1d0680-4b24-11e9-9f48-ae339aaca114.png)